common/build-helper/rust.sh

```diff
@@ -67,3 +67,6 @@ export RUSTONIG_SYSTEM_LIBONIG=1
 
 # libsqlite3-sys
 export LIBSQLITE3_SYS_USE_PKG_CONFIG=1
+
+# libgit2-sys
+export LIBGIT2_NO_VENDOR=1
```

Comment:
i think we were going to hold off on this until the libgit2 update

Reply:
I've removed it for now.